Remove -fdebug-prefix-map=/p/w/d=. from CFLAGS and CONFIGURE in pg_config. The
embedded path makes the build non-reproducible, and the flag is useless for
building extensions anyway.

--- a/src/common/Makefile
+++ b/src/common/Makefile
@@ -26,10 +26,10 @@ LIBS += $(PTHREAD_LIBS)
 # don't include subdirectory-path-dependent -I and -L switches
 STD_CPPFLAGS := $(filter-out -I$(top_srcdir)/src/include -I$(top_builddir)/src/include,$(CPPFLAGS))
 STD_LDFLAGS := $(filter-out -L$(top_builddir)/src/port,$(LDFLAGS))
-override CPPFLAGS += -DVAL_CONFIGURE="\"$(configure_args)\""
+override CPPFLAGS += -DVAL_CONFIGURE="\"$(filter-out -fdebug-prefix-map%,$(configure_args))\""
 override CPPFLAGS += -DVAL_CC="\"$(CC)\""
 override CPPFLAGS += -DVAL_CPPFLAGS="\"$(STD_CPPFLAGS)\""
-override CPPFLAGS += -DVAL_CFLAGS="\"$(CFLAGS)\""
+override CPPFLAGS += -DVAL_CFLAGS="\"$(filter-out -fdebug-prefix-map%,$(CFLAGS))\""
 override CPPFLAGS += -DVAL_CFLAGS_SL="\"$(CFLAGS_SL)\""
 override CPPFLAGS += -DVAL_LDFLAGS="\"$(STD_LDFLAGS)\""
 override CPPFLAGS += -DVAL_LDFLAGS_EX="\"$(LDFLAGS_EX)\""
